### DISCUSSION TITLE
not ok 126 - getnameinfo_basic_ip6

### DISCUSSION THREAD

**Discussion by bhanu-atturu** [libuv/libuv#3442] [Q&A]
The test getnameinfo_basic_ip6 (using libuv v1.38.0) is failing on ubuntu18.04, centos6 with restricted network with the following output:

```
# exit code 6
# Output from process `getnameinfo_basic_ip6`:
# Assertion failed in test/test-getnameinfo.c on line 42: status == 0
```

We have a DNS server which will allow traffic only to predesignated domain names. The test passes on centos7 and rhel8. If we remove the external traffic restriction, then the test passes on all OS. Is there any external domain that is required for this test?

**Comment by bnoordhuis** [comment]
Not an external domain - that test expects a reverse lookup for ::1 to resolve to something, doesn't really matter what, as long as it doesn't fail.

**Reply by bhanu-atturu** [reply]
Thanks for the quick reply! After whitelisting the domain ip6.arpa, the test is passing.
